package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fusebit/everyauth-express",
-  "version": "1.0.4",
+  "version": "2.0.0",
```

Comment:
This isn't a breaking change, is it? Would minor be sufficient?

Reply:
It changes expectations around the JSON-format of the profile in EVERYAUTH_PROFILE_JSON, unfortunately, because it previously wasn't using the 'export/import' format that the CLI generates.